department-of-veterans-affairs/va.gov-cms
# VACMS-3026: Audit VAMC top task pages.

## Description

See #3026. 

## Testing done

Manual, behat

## Screenshots

![VAMC_top_task_pages___VA_gov_CMS](https://user-images.githubusercontent.com/643678/95085093-c27db980-06ec-11eb-8956-280f0cb0e58f.jpg)

## QA steps

As content_admin or admin, go to admin/content/audit/vamc-top-task-pages

You should see the all VAMC detail pages with the following URLs

/medical-records-office
/register-for-care
/billing-insurance
/policies
/make-an-appointment
/contact-us
/pharmacy


## Definition of Done
